specifyweb/specify/migrations/0004_schema_config_update.py

```diff
@@ -0,0 +1,177 @@
+"""
+This migration updates the Schema Config entries for pre-geo tables. 
+Geo migration added new fields to already existing tables. This migration updates the Schema Config to reflect those changes.
+
+Fields added:
+Collection -> collectionObjectType
+GeographyTreeDef -> discipline
+GeologicTimePeriodTreeDef -> discipline
+LithostratTreeDef -> discipline
+StorageTreeDef -> institution
+TaxonTreeDef -> discipline
+"""
+from django.db import migrations
+
+FIELD_DATA = [
+    {
+        "table": "Collection",
+        "field": "collectionObjectType",
+        "isrequired": False,
+    },
+    {
+        "table": "GeographyTreeDef",
+        "field": "discipline",
+        "isrequired": True,
+    },
+    {
+        "table": "GeologicTimePeriodTreeDef",
+        "field": "discipline",
+        "isrequired": True,
+    },
+    {
+        "table": "LithostratTreeDef",
+        "field": "discipline",
+        "isrequired": True,
+    },
+    {
+        "table": "StorageTreeDef",
+        "field": "institution",
+        "isrequired": True,
+    },
+    {
+        "table": "TaxonTreeDef",
+        "field": "discipline",
+        "isrequired": True,
+    },
+]
+
+PICKLIST_NAME = 'CollectionObjectGroupType'
+COGTYPE_FIELD_NAME = 'cogType'
+PICKLIST_TEXT = 'Collection Object Group Type'
+
+def add_fields(apps):
+    Splocalecontainer = apps.get_model('specify', 'Splocalecontainer')
+    Splocalecontaineritem = apps.get_model('specify', 'Splocalecontaineritem')
+    Splocaleitemstr = apps.get_model('specify', 'Splocaleitemstr')
+
+    for data in FIELD_DATA:
+        for container in Splocalecontainer.objects.filter(name=data['table'], schematype=0):
+            container_item, _ = Splocalecontaineritem.objects.get_or_create(
+                name=data["field"],
+                type='ManyToOne',
+                container=container,
+                isrequired=data["isrequired"]
+            )
+            Splocaleitemstr.objects.get_or_create(
+                language='en',
+                text=data["field"],
+                itemname=container_item
+            )
+            Splocaleitemstr.objects.get_or_create(
+                language='en',
+                text=data["field"],
+                itemdesc=container_item
+            )
+
+def remove_fields(apps):
+    Splocalecontaineritem = apps.get_model('specify', 'Splocalecontaineritem')
+    Splocaleitemstr = apps.get_model('specify', 'Splocaleitemstr')
+
+    for data in FIELD_DATA:
+        Splocaleitemstr.objects.filter(
+            text=data["field"].title(),
```

Comment:
Is this `.title()` intentional? Or did you mean to include a `.title()` when creating the `Splocaleitemstr`? 
The `text` here might be different from creating the Splocaleitemstr.

Reply:
I think I intended to include it when creating a Splocaleitemstr but I'll get rid of it just to keep things simpler